src/transformers/models/bit/configuration_bit.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" BiT model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+BIT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "google/resnetv2-50": "https://huggingface.co/google/resnetv2-50/resolve/main/config.json",
```

Comment:
This doesn't look like a BiT checkpoint, could we make sure it's present in the name?

Reply:
Fixed in [93669f7](https://github.com/huggingface/transformers/pull/20550/commits/93669f7fb53bbdc56a73eb8b4c9cae553ad8cf94)